interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -628,6 +634,48 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  //-----------------------------------------------------------------------------
+  // The tests below are only work with gRpc interop test framework.
+  //-----------------------------------------------------------------------------
+
+  /**  Sends a large unary rpc with service account credentials. */
+  public void serviceAccountCreds(String jsonKey, InputStream credentialsStream, String authScope)
+      throws Exception {
+    ServiceAccountCredentials credentials =
```

Comment:
The type here can just be GoogleCredentials, right?

You could still do the cast, and comment that you are just double-checking that the right type of object was created.